sub/import.js

```diff
@@ -14,10 +14,26 @@ module.exports = function (argv) {
   var charactersDirectory = 'persist/characters';
   var dest = path.join(charactersDirectory, filename);
 
-  if (!fs.existsSync(charactersDirectory)) {
-    fs.mkdirSync(charactersDirectory);
-  }
-
+  /*
+  "fs.existsSync(path)" has been depreciated.  It's probably a good idea to shift
+  this to the more current 'fs.accessSync(path, fs.F_OK)'.
+  OLD CODE:
+  ---
+    if (!fs.existsSync(charactersDirectory)) {
+      fs.mkdirSync(charactersDirectory);
+    }
+  ---
+  */
+  //--- new code
+  //fs.R_OK checks to see if the file can be read and fs.W_OK checks if it can be written
+  //the previous version only
+    fs.access(charactersDirectory, fs.F_OK, function(err) {
```

Comment:
Since you use an async method here, there's no guarantee that the existence check will return before we create the readStream.  Use accessSync